inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/InlongGroupServiceImpl.java

```diff
@@ -421,4 +434,38 @@ public void saveOrUpdateExt(String groupId, List<InlongGroupExtInfo> infoList) {
         LOGGER.info("success to save or update inlong group ext for groupId={}", groupId);
     }
 
+    @Override
+    public InlongGroupTotalMetricResponse getMetric(InlongGroupMetricRequest request) {
```

Comment:
Please refer to the `HeartbeatController`, and re-consider the design of those interfaces.